### ISSUE TITLE
Latest published version to npm is missing source

### ISSUE THREAD

**Issue by MylesBorins** [stefanpenner/promise.hash.helper#7]
Only seeing a package.json and README.md. This is breaking ember-cli in CITGM (node.js smoke test).

To reproduce 

```
$ npm pack promise.hash.helper && tar -zxf promise.hash.helper-1.0.3.tgz && ls package/
```

**Comment by stefanpenner** [comment]
😱

**Comment by stefanpenner** [comment]
Thanks for the report, this was fixed with 1.0.4 release + commit https://github.com/stefanpenner/promise.hash.helper/commit/49a935479a0d64c8698462042776d6737bb8ede3
